HackYourFuture/React
# Adham week3 react homework

**Homework React week3**
I made 2 fetch in the code. 
First form origin source  to get list (it is not working in the online version. Netlify dose't support running `server.js`). 
Second to a remote server to get UserInfo.

**To test it:** 

- Clone or download the repo .

- Go to the directory in the terminal  :`homework-submission\week3_app` and run `npm install`

- In the same directory run this script: `npm run build`  because I am serving the HTML in the Build Folder. So you can access it later through : `http://localhost:8080/` 

- In the same directory run this script in terminal:  `npm run online` , This will runs the `server.js` and the `ReactApp` . Then wait few minutes to the server and the React app will run.  .

you can see the static version on this link: https://inspiring-leakey-59e976.netlify.com

I know that what I have done is not exactly what they ask in the homework. But Fetching data and then connecting my server with React App  was something very  interessant to me, so I went to the wrong destination.  

### Thanks
